core/tests/infrastructure/test_zip.py

```diff
@@ -0,0 +1,24 @@
+from cat.infrastructure.package import Package
+import os
+
+
+def test_unpackage():
+    zip = Package("./tests/infrastructure/plugin.zip")
```

Comment:
No need for zip files in the repo. Please create a zip  of the `mock_plugin` via code, as done [here](https://github.com/cheshire-cat-ai/core/blob/f579b4badd942bb6862a055fcebb959db788d3c5/core/tests/routes/test_routes_plugins.py#L44)